hydrogram/methods/phone/invite_group_call_members.py

```diff
@@ -0,0 +1,91 @@
+#  Hydrogram - Telegram MTProto API Client Library for Python
+#  Copyright (C) 2017-2023 Dan <https://github.com/delivrance>
```

Comment:
You don't have to grant Dan the copyright to a file that wasn't part of Pyrogram before the rebranding to Hydrogram.